content/en/blog/2021/istio-community-meetup-china/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "The first local Istio meetup is here!"
+description: "Join Istio community experts in China."
+publishdate: 2021-07-06
+attribution: "Iris Ding (Intel), Maria Cruz (Google)"
+keywords: [community,meetup]
+---
+
+With the popularization of cloud native technology in China, Istio has also gained popularity in this corner of the world in the past two years. Almost all Chinese CSPs are creating and running their service mesh products based on Istio. Since the first IstioCon held in February 2021, the project has been widely welcomed by developers. Developers have also expressed an interest in participating in more Istio meetups and helping to grow the community at the local level. This is an opportunity to share and exchange across the globe.
```

Comment:
```suggestion
With the rapid popularization of cloud native technology in China, Istio has also gained popularity in this corner of the world. Almost all Chinese CSPs have creating and are running service mesh products based on Istio. 

We welcomed thousands of Istio users and developers to the first IstioCon in February 2021, and the attendees expressed an interest in participating in more meetups and helping to grow the community at the local level.
```

removed "This is an opportunity to share and exchange across the globe." as it's not really relevant in a China-only contex.t